src/global_scheduler/global_scheduler.cc

```diff
@@ -14,6 +14,45 @@
 #include "state/task_table.h"
 
 /**
+ * Retry the task assignment. If the local scheduler that the task is assigned
+ * to is no longer active, do not retry the assignment.
+ * TODO(rkn): We currently only retry the method if the global scheduler
+ * publishes a task to a local scheduler before the local scheduler has
+ * subscribed to the channel. If we enforce that ordering, we can remove this
+ * retry method.
+ *
+ * @param id The task ID.
+ * @param user_context The global scheduler state.
+ * @param user_data The Task that failed to be assigned.
+ * @return Void.
+ */
+void assign_task_to_local_scheduler_retry(UniqueID id,
```

Comment:
If we make some similar changes to the local scheduler then I think that will solve #807.

Reply:
With the last couple commits, the error messages will still get spewed while the actor is getting timed out, but will stop after the actor is marked as dead, instead of continuing forever. We can hide the error messages (possibly hiding actual errors later on), or we can retry the `task_table_add` after a timeout (possibly adding some unnecessary latency). Neither of these seems desirable.

Reply:
So the behavior after this PR is that it logs excessively for 10 seconds and then it stops, right? That's a big improvement over logging excessively forever.